src/utils/convertToCsv.ts

```diff
@@ -101,6 +93,10 @@ export const convertToCSV = (data: any) => {
           value = value.substring(1, value.length - 1)
         }
 
+        if (value.includes("'")) {
+          value = '"' + value + '"'
+        }
+
         value = value.replace(/\\"/gm, '""')
```

Comment:
shouldn't this be _above_ the previous line?  Else we will double double quote the wrapping quotes

Reply:
Does that mean that this is not the correct output for the input below?

```
foo:$7. bar:$9.\r\n"""bar""",abc\r\nsadf,def\r\nasd,"""qwert"""
```

Input:
```
[
      { foo: `"bar"`, bar: 'abc' },
      { foo: 'sadf', bar: 'def' },
      { foo: 'asd', bar: `"qwert"` }
]
```

Reply:
The test case shown doesn't have single quotes - it would seem that single quotes will result in quadrupling the double quotes

Reply:
Given:
```
'blah'
"blah"
'blah,"'
"blah,blah" "
',''
","
','
,",
"
'
,
',
,'
",
,"
```
Expected:
```
"'blah'"
"""blah"""
"'blah,""'"
"""blah,blah"" """
"',''"
""","""
"','"
","","
""""
"'"
","
"',"
",'"
""","
","""
```

Reply:
This is the test case for single quotes, which is now passing.

```
const data = [
      { foo: `'bar'`, bar: 'abc' },
      { foo: 'sadf', bar: 'def' },
      { foo: 'asd', bar: `'qwert'` }
    ]

    const expectedOutput = `foo:$5. bar:$7.\r\n"'bar'",abc\r\nsadf,def\r\nasd,"'qwert'"`

```